### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by shawnnicolekim**
### 1. Summary of Changes
This PR deletes the original homepage/landing page and now routes any homepage calls to the "Map" tab of the navigation bar. A new "Get Started" tab has been added to the navigation bar that contains directions on how to use the Energy Dashboard, as well as FAQ's. A message box will direct users to the "Get Started" tab if it's their first time using the Energy Dashboard.

### 2. Itemized List of Changes:

**Got rid of original landing page:**
- Deleted entire `home` folder
- `src/router/index.js`: Homepage URL or clicking on top-left OSU logo will redirect to "Map" tab. Added "/getstarted" route.
- Added new "get_started" folder in `src/components`

**get_started folder contains:**
- `buildingListTab.vue`: contains steps and image listed under "How To Use The "Building List" Tab"
- `commitment.vue`: kept this file from the previous `home` folder.
- `faq.vue`: contains all of the FAQ questions and answers. User should be able to open and close all collapsible items.
- `feedbackForm.vue`: this file is not being used right now, since we're using a Google Form instead, but I'd like to save it for future use when we decide to build out our own Feedback Form.
- `getStartedBanner.vue`: this file uses the `commitment.vue` file to create the entire image banner
- `getStartedContent.vue`: this file contains all of the other components (`getStartedBanner.vue`, `faq.vue`, `mainDescription.vue`) to create the entire "Get Started" page.
- `mainDescription.vue`: contains everything between the image banner to the FAQ's, such as the dividers, `mapTab.vue`, and `buildingListTab.vue`.
- `mapTab.vue`: listed under the "How To Use The "Map" Tab", this contains tabs of steps and gifs. User should be able to click between the different tabs. 

**Created a cookie for first timer's:**
- Implemented the `mounted()` function in `App.vue ` that will pop up a message box if it's the user's first time to the site. If the user selects to never see the message again, it will create a cookie so that the message never shows for 10 years.

### CODE REVIEW

File: src/components/get_started/getStartedBanner.vue [hunk 1]
```diff
@@ -1,37 +1,54 @@
...
+  padding-top: 1em;
+  padding-left: 10px;
+  background: url('/images/solar.jpg');
+  background-size: cover;
+  background-repeat: no-repeat;
 }
+
+/*
+* Don't think I need this
+
```
Comment (jackrwoods) [inline src/components/get_started/getStartedBanner.vue:51]: We could just delete the commented-out code. Git will keep a history of it if we want to refer to it later.

**Review by jackrwoods — APPROVED** [review]

File: src/components/get_started/mainDescription.vue [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    }
+  }
+}
+</script>
+
+<style>
+.divider {
+  background-color: #D73F09;
+  margin-top: 40px;
+  margin-bottom: 40px;
```
Comment (MilanDonhowe) [inline src/components/get_started/mainDescription.vue:41]: We may want to increase the font-size of the dividers in the future to make its internal text stand out a little more.

File: src/components/get_started/mapTab.vue [hunk 3]
```diff
@@ -0,0 +1,110 @@
+<template>
+  <el-tabs type="card" v-model="mapTab" >
```
Comment (MilanDonhowe) [inline src/components/get_started/mapTab.vue:2]: This looks awesome--great usage of the `el-tab`s.

**Review by MilanDonhowe — APPROVED** [review]
looks good!

**Pull request merged**